spec/factories/state_file/state_file_archived_intake_requests.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :state_file_archived_intake_request do
+    email_address { "geddy_lee@gmail.com" }
+    failed_attempts { 0 }
+    locked_at { nil }
+
+    trait :locked do
```

Comment:
🤔 does it make sense to put this in the past? or does it have to be the current time, considering that sometimes when tests run on CI, current time can be an issue.